### ISSUE TITLE
Support 0.14 of React

### ISSUE THREAD

**Issue by justin808** [shakacode/react_on_rails#51]
Simple change to gem:

1. Change docs to expose ReactDom like this:
      // React is necessary for the client rendering:
      {test: require.resolve('react'), loader: 'expose?React'},
2. Add function to resolve correct "React" object: 

Here's where we refer to React right now.

https://github.com/shakacode/react_on_rails/blob/master/app/assets/javascripts/react_on_rails.js#L21
https://github.com/shakacode/react_on_rails/blob/master/app/assets/javascripts/react_on_rails.js#L43

Ideally, have some sort of way to test this.

Need to adjust docs.

**Comment by justin808** [comment]
@rstudner has volunteered to give this a try.

**Comment by rstudner** [comment]
Quick update.  It is all working.  I'll make a separate PR for updating the react-webpack-rails-tutorial project to use 0.14 also just to update the package.json to 0.14 (or someone else can just do that, once the gem is approved.

There are still some warning etc in the console i'm tracking down -- i've seen some postings they may be related to react-redux/not our concern just yet.

<img width="1096" alt="screen shot 2015-10-08 at 7 35 16 pm" src="https://cloud.githubusercontent.com/assets/209970/10382836/25fd64e6-6df6-11e5-9155-288c23779d19.png">

**Comment by justin808** [comment]
Thanks so much for the help! :+1: 

https://github.com/shakacode/react-webpack-rails-tutorial/pull/115 is merged.

**Comment by justin808** [comment]
Fixed in #55
